lib/Doctrine/Migrations/Version/ExecutionResult.php

```diff
@@ -49,57 +50,40 @@ class ExecutionResult
     /** @var Schema|null */
     private $toSchema;
 
-    /**
-     * @var Version
-     */
+    /** @var Version */
     private $version;
 
-    /**
-     * @var string
-     */
+    /** @var string */
     private $direction;
```

Comment:
Is it worth to have a value Direction with Direction::up() and Direction::down() in general?

Reply:
I would say yes. It avoids typos and makes clear that there are two possible values without having to look in the abstract migration class.

Here is same debate of enums vs plain strings imo

Reply:
Glad to hear that